apps/chat/src/components/AppsEditor/AppsEditorHeader.tsx

```diff
@@ -177,7 +184,10 @@ export const AppsEditorHeader: React.FC<AppsEditorHeaderProps> = ({
             </button>
           ) : (
             <Link
-              className="hidden items-center space-x-1 hover:text-accent-primary md:flex"
+              className={classNames(
+                'hidden items-center space-x-1 hover:text-accent-primary md:flex',
+                isMindmapSettingsPage && 'text-accent-primary',
+              )}
```

Comment:
I tnink only this line cnage will be enough
```suggestion
              className="hidden items-center space-x-1 text-accent-primary md:flex"
```